sdk/appcenter/src/main/java/com/microsoft/appcenter/channel/DefaultChannel.java

```diff
@@ -761,7 +775,22 @@ synchronized void checkPendingLogs(@NonNull GroupState groupState) {
             triggerIngestion(groupState);
         } else if (pendingLogCount > 0 && !groupState.mScheduled) {
             groupState.mScheduled = true;
-            mAppCenterHandler.postDelayed(groupState.mRunnable, groupState.mBatchTimeInterval);
+            long delay = groupState.mBatchTimeInterval;
+            if (groupState.mBatchTimeInterval > MINIMUM_TRANSMISSION_INTERVAL) {
+                long now = System.currentTimeMillis();
+                long startTimer = SharedPreferencesManager.getLong(START_TIMER_PREFIX + groupState.mName);
+                if (startTimer == 0) {
+                    SharedPreferencesManager.putLong(START_TIMER_PREFIX + groupState.mName, now);
+                } else if (startTimer > now) {
```

Comment:
what is this case?

Reply:
https://github.com/microsoft/appcenter-sdk-android/pull/1165/files#diff-0ebb2cb4967c7aee91e0b189502daf32R786

Reply:
The comment should be in this PR instead of the other one.